### DISCUSSION TITLE
how to solve KeeperErrorCode = ConnectionLoss

### DISCUSSION THREAD

**Discussion by XY-CHEN-DB** [apache/kyuubi#2799] [Q&A] [answered]
org.apache.kyuubi.shade.org.apache.curator.ConnectionState:228  - Connection timed out for connection string (192.168.138.156:2181) and timeout (15000) / elapsed (15997)
org.apache.kyuubi.shade.org.apache.curator.CuratorConnectionLossException: KeeperErrorCode = ConnectionLoss

this exception will take lost of time , thougth finally the query will execute. how to slove it

**Comment by yaooqinn** [comment, marked as answer]
it seemed the network between zk cluster and clients is not stable

**Comment by XY-CHEN-DB** [comment]
您好，邮件已收到

**Reply by XY-CHEN-DB** [reply]
感谢你的回复，这个对的。
可能我在模拟时候，配置太低导致缓慢。
后来通过配置属性
System.setProperty("curator-default-connection-timeout","45000");
来解决这个异常提示。当在配置稍微高点的机器模拟时，完全没有出现这个问题。

**Reply by yaooqinn** [reply]
谢谢确认
